src/NHibernate/NHibernateUtil.cs

```diff
@@ -242,6 +242,11 @@ public static IType GuessType(System.Type type)
 		
 		public static readonly NullableType DbTimestamp = new DbTimestampType();
 
+        /// <summary>
+        /// NHibernate timestamp utc type.
+        /// </summary>
+        public static readonly NullableType TimestampUtc = new TimestampUtcType();
```

Comment:
Can you please change this to `public static readonly TimestampUtcType TimestampUtc = new TimestampUtcType()`;?